src/compiler/diagnosticMessages.json

```diff
@@ -1439,6 +1439,14 @@
         "category": "Error",
         "code": 2500
     },
+    "Duplicate identifier '_arguments'. Compiler uses variable declaration '_arguments' to capture 'arguments' reference.": {

```

Comment:
Can we just change these to be templated text, and pass the relevant identifiers?